### ISSUE TITLE
If item does not exist, GivePlugin won't return error

### ISSUE THREAD

**Issue by williammck** [terrapinio/snappy#1]
need to add a constant of items?

not really sure about what the best way to do this is. @barneygale

**Comment by barneygale** [comment]
yeah I think we need to look up items outselves and never run vanilla /give with an item id that doesn't exist
